tests/test.py

```diff
@@ -124,11 +125,14 @@ def multiple_warnings(self, warning_count=1, error=False):
 def tuberd(pytestconfig):
     """Spawn (and kill) a tuberd"""
 
-    tuberd = "tuberd"
```

Comment:
Why not always the "-m tuber.server" path?

Reply:
The `entry_points` option to `setup()` installs a `tuberd` executable (which just does `from tuber.server import main; main()`), and I wanted to test that it was getting installed properly and ending up on the path that the tests were using.